Jenkinsfile

```diff
@@ -100,7 +100,7 @@ pipeline {
                 withMaven(maven: 'maven-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
                     sh '''
                         unset JAVA_TOOL_OPTIONS
-                        mvn install -B -pl $ITESTS -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
+                        mvn install -B -pl $ITESTS,!$ITCORE -amd -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
```

Comment:
This should be fine. At one point we considered splitting core and itests into separate stages so that we could track stability between the set of tests a little easier.

Reply:
Oh thanks for pointing that out. I copied these changes from master, where the stages _are_ separate. But that must have been changed after the 2.26.x branch was made, because they're all run in one step here.

I'll need to correct this since $ITCORE isn't even defined in this Jenkinsfile

Reply:
Oh yeah good call. I totally missed it wasn't defined here, but we would have figured that out fast :-)